sqlmesh/dbt/loader.py

```diff
@@ -102,6 +102,8 @@ def _load_models(
                 }
             )
 
+        models.update(self._load_external_models())
```

Comment:
Shouldn't `_load_external_models` be called in the `load` directly and then we wouldn't need to call it explicitly here?

Reply:
Yeah, I can do it that way. It'd still look like `models.update(super()._load_models(args))`, so I thought the code was simpler by duplicating the one line, but I can use inheritance if you prefer. Let me know.